tasks/darwin/disable-service.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Include platform-specific defaults
+  include_vars: "{{ role_path }}/vars/{{ ansible_os_family | lower }}.yml"
+
+- name: Get current UID
+  command: "id -u"
+  changed_when: false
+  register: uid
+
+- name: Get list of disabled services
+  command: "launchctl print-disabled gui/{{ uid.stdout }}"
+  changed_when: false
+  register: disabled_services
+
+- name: Show value of disabled_services
+  debug:
+    var: disabled_services
+
+- name: Show value of jenkins_swarm_client_disabled
+  debug:
+    var: jenkins_swarm_client_disabled
+
+
+- name: Disable Jenkins swarm client
+  command: "launchctl disable gui/{{ uid.stdout }}/{{ jenkins_swarm_client_ident }}"
+  when: jenkins_swarm_client_disabled not in disabled_services.stdout_lines
```

Comment:
No, because if a service is disabled and then re-enabled, it shows up in the output as `"com.company.ident" => false`, so we need to also match against the boolean. TBH, I'm not sure why such things are even listed in this output, but alas, they are. :shrug: